timeless/auth/views.py

```diff
@@ -53,14 +53,12 @@ def forgot_password():
     if request.method == "POST":
         email = request.form["email"]
         error = auth.forgot_password(email=email)
-        print(error)
-        if error is not None:
+
+        if error:
             return render_template("auth/forgot_password.html", error=error)
-        else:
-            return render_template(
-                "auth/forgot_password_post.html",
-                email=email
-            )
+
+        return render_template(
+            "auth/forgot_password_post.html", email=auth.mask_email(email))
```

Comment:
@vryazanov since you are working on this code, could we do something like:

```python
template; 
if error:
    template = render_template(..., error);
else: 
    template = render_template(..., email);
return template
```

So, a single return statement. It always seemed more clear to me :D

Reply:
@amihaiemil ok, I will fix it :+1: